src/NHSD.BuyingCatalogue.Organisations.Api/Repositories/OrganisationRepository.cs

```diff
@@ -26,5 +26,14 @@ public async Task<Organisation> GetByIdAsync(Guid id)
         {
             return await _context.Organisations.FirstOrDefaultAsync(org => org.OrganisationId == id);
         }
+
+        public async Task UpdateAsync(Organisation organisation)
+        {
+            if (organisation is null) 
```

Comment:
Not sure if this check is needed as you are checking if it is null before-hand.

Reply:
I was doing the check with future uses in mind (this method might get used somewhere else and I can't guarantee that the user will check for null). If you still feel like that's being overly cautious let me know and I'll change it :]

Reply:
No you are perfectly right tbh.